### ISSUE TITLE
largeIcon exception on notification displayed Android

### ISSUE THREAD

**Issue by AndrewMorsillo** [notifee/react-native-notifee#139]
When displaying a notification on Android I see the following in logcat

```
 E/NOTIFEE: (NotificationManager): An error occurred whilst trying to retrieve a largeIcon image: @mipmap/ic_launcher
    java.lang.NullPointerException: ImagePipelineFactory was not initialized!
        at com.facebook.common.internal.Preconditions.checkNotNull(Preconditions.java:231)
        at com.facebook.imagepipeline.core.ImagePipelineFactory.getInstance(ImagePipelineFactory.java:69)
        at com.facebook.drawee.backends.pipeline.Fresco.getImagePipelineFactory(Fresco.java:140)
        at com.facebook.drawee.backends.pipeline.Fresco.getImagePipeline(Fresco.java:145)
        at n.o.t.i.f.e.e.nZ_ne.nz_n(SourceFile:54)
        at n.o.t.i.f.e.e.nz_f.nz_n(SourceFile:161)
        at n.o.t.i.f.e.e.-$$Lambda$QFUpjtf8TZZqJ7Ktkw5KoUu43uQ.then(Unknown Source:2)
        at com.google.android.gms.tasks.zze.run(com.google.android.gms:play-services-tasks@@17.1.0:5)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:923)
```

The icon seems to work fine however.

**Comment by helenaford** [comment]
Hi, thanks for reporting the bug. I've looked into it and can reproduce this locally. Will follow-up again shortly with more info, and hopefully a fix.

**Comment by helenaford** [comment]
Fix is in the latest release (v0.15.1) 👍
